src/astx/operators.py

```diff
@@ -128,3 +129,49 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = str(self)
         value = self.value.get_struct(simplified)
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class AugAssign(DataType):
+    """AST class for augmented assignment."""
+
+    target: Identifier
+    op_code: Literal[
+        "+=",
+        "-=",
+        "*=",
+        "/=",
+        "//=",
+        "%=",
+        "**=",
+        "&=",
+        "|=",
+        "^=",
+        "<<=",
+        ">>=",
+    ]
+    value: DataType
+
+    def __init__(
+        self,
+        target: Identifier,
+        op_code: str,
```

Comment:
the op_code in the class scope is a literal, not a str, so it should be a literal as well.

Instead of defining the Literal type twice, please create at the beginning of the file a type alias called **OpCodeAugAssign** with the proper value (Literal[...]) and reuse it.

Take a look into this as a reference: https://github.com/arxlang/astx/blob/main/src/astx/base.py#L420